searx/engines/freesound.py

```diff
@@ -0,0 +1,68 @@
+"""
```

Comment:
Can you add a .

    # SPDX-License-Identifier: AGPL-3.0-or-later

Reply:
sure